xwiki-platform-core/pom.xml

```diff
@@ -138,10 +138,23 @@
 
                  Single justification example:
             -->
-            
-            
-            
-            
+
+            <revapi.differences>
+              <justification>Change in generated class of the REST model to add the
+                enforceRequiredRights property.</justification>
+              <criticality>highlight</criticality>
```

Comment:
Feels like an `allowed` to me (I don't see what this change could really break in practice).

Reply:
Okay, I've changed it, thank you!